docs/platforms/dart/features/index.mdx

```diff
@@ -20,3 +20,4 @@ Sentry's Dart SDK enables automatic reporting of errors and exceptions, and iden
   - [File I/O Integration](/platforms/dart/integrations/file/).
 - [Logging Integration](/platforms/dart/integrations/logging).
 - [Dio Integration](/platforms/dart/integrations/dio/).
+- [Metrics](/product/performance/metrics/) help you track custom data points to gain insights into app performance and user experience
```

Comment:
```suggestion
- [Metrics](/product/performance/metrics/) help you track custom data points to gain insights into app performance and user experience.
```
Hey there, just want to double check that you want this to point to performance metrics not these [metrics](https://docs.sentry.io/product/explore/metrics/)...